### PR TITLE
texlive update

### PR DESCRIPTION
**Pull request by gzagatti**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

This PR successfully install `texlive` and improves path handling and post installation caveats. 

The previous formula used a `portable` install which did not correctly created the symlinks necessary to operate with `brew`. This PR addresses this issue. 

The new caveat message also instruct the user on how to update the path after installing new texlive packages if one so wishes.

### CODE REVIEW

**Comment by gzagatti** [comment]
I have no idea why the CI is failing again. It fails to update because it claims that `Homebrew` is running as root. I think this is an issue with CI.

File: Formula/texlive.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 class Texlive < Formula
   desc "TeX Live is a free software distribution for the TeX typesetting system"
   homepage "https://www.tug.org/texlive/"
-  url "http://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz"
-  version "20190406"
-  sha256 "c7742ea5b0bc22fe2742e9fa2bf9aeb8ff88175722fcfb2b72c00a29c06e2fc9"
+  url "https://www.texlive.info/tlnet-archive/2020/07/15/tlnet/install-tl-unx.tar.gz"
+  version "20200715"
+  sha256 "517058e56756521c3ab1b1939e5e95659adc715ba27babdff41b96bd299e3d20"
+  revision 1
```
Comment (iMichka) [inline Formula/texlive.rb:7]: We do not need a revision if we update the version

File: Formula/texlive.rb [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+  head "http://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz"
 
   bottle do
     cellar :any_skip_relocation
```
Comment (iMichka) [inline Formula/texlive.rb:11]: Please remove the linux sha256 bottle line.

File: Formula/texlive.rb (hunk 1 above)
Comment (gzagatti) [reply Formula/texlive.rb:7]: Oh right. So what do I do with the bottle section below? It seems like CI is failing because of that.

**Comment by iMichka** [comment]
Please add the xorg libraries back, texlive links against them:

```
==> brew linkage --test texlive
==> FAILED
Missing libraries:
  libGL.so.1
  libICE.so.6
  libSM.so.6
  libX11.so.6
  libXaw.so.7
  libXext.so.6
  libXmu.so.6
  libXpm.so.4
  libXt.so.6
  libglut.so.3
Unwanted system libraries:
  /lib/x86_64-linux-gnu/libncurses.so.5
  /lib/x86_64-linux-gnu/libtinfo.so.5
```

**Comment by iMichka** [comment]
Nice work. CI is green. I squashed your commits to a single one and merged it. Thanks for the improvements!

**Comment by gzagatti** [comment]
Thanks! That was my first contribution to Homebrew!

**Pull request merged**